### DISCUSSION TITLE
AraBERT-Trainer-HyperParameterOpt-NER database error

### DISCUSSION THREAD

**Discussion by Sharefah-Alghamdi** [aub-mind/arabert#131] [Q&A] [answered]
Hi everyone!

I'm trying to run the code for fine-tuning of Arabert for NER code here:

[https://colab.research.google.com/drive/1M1yH8rqbxowlSd5R-jr8eBVfGbIyp1zC?usp=sharing](url)

but I need to understand what is the needed database for storage in this part:

best_run = trainer.hyperparameter_search(direction="maximize",
                                         hp_space=my_hp_space,
                                         compute_objective=my_objective,
                                         n_trials=None,
                                         pruner=optuna.pruners.NopPruner(),
                                         sampler=optuna.samplers.GridSampler(search_space),
                                         study_name="NER-arabert-large-v02",
                                         storage="sqlite:////content/drive/MyDrive/optuna_runs/{}.db".format(name),
                                         load_if_exists=False
                                         )

Thank you

**Comment by WissamAntoun** [comment, marked as answer]
This is used for hyper-parameter tunning using optuna, the sql db is used to store the metrics and hyperparams. 

If you are not doing hyper-parameter tuning you can just ignore it
